tests/compiler/compiler.vitest.test.ts

```diff
@@ -252,6 +252,10 @@ describe('compiler', () => {
       export var root = 'styles_a_root__mh4uy80 shared_shared__5i7sy00';"
     `);
 
+    // styles_a.css.ts should not generate any CSS
+    const { css: cssA } = compiler.getCssForFile(cssPathA);
+    expect(cssA).toBe('');
+
```

Comment:
I'm not seeing the failure when I run the tests locally. This doesn't surprise me, as this line ensures the returned `css` is at least an empty string (`''`).

https://github.com/vanilla-extract-css/vanilla-extract/blob/837774c13514c94f46796a44cfa9da8451d80974/packages/integration/src/compiler.ts#L322

Reply:
Same. I think this probably isn't the correct thing to test. We want to test that the file isn't even processed, rather than it doesn't generate any CSS (it never will).

Reply:
Removed the assertions.